dev_env_setup.sh

```diff
@@ -12,145 +12,75 @@ cd "$SCRIPT_DIR"
 echo "Running 'yarn install' in the root directory..."
 yarn install
 
-# Step 2: Run yarn build in the root directory
-echo "Running 'yarn build' in the root directory..."
-yarn build
-
-# Step 3: Run yarn build in packages/cubejs-playground
-echo "Running 'yarn build' in packages/cubejs-playground..."
-cd packages/cubejs-playground
-yarn build
-cd ../..
-
-# Step 4: Run yarn tsc for the first time
-echo "Running 'yarn tsc --build'..."
-yarn tsc
-
-# Step 5: List available drivers and ask user to select
-echo "Listing available drivers..."
-available_drivers=$(ls packages | grep "driver")
-
-PS3='Please select the drivers you want to use (enter number, then press Enter): '
+# Step 2: Build all packages in packages directory
+echo "Building all packages..."
+for package in packages/*; do
+    if [ -d "$package" ]; then
+        echo "Building $package..."
+        cd "$package"
+        if ! yarn build; then
+            echo "yarn build failed for $package, trying yarn tsc..."
+            yarn tsc || true  # Continue even if tsc fails
+        fi
+        cd "$SCRIPT_DIR"
+    fi
+done
 
-# Display drivers without the prefix "cubejs-"
-select selected_driver in $(echo "$available_drivers" | sed 's/cubejs-//') "Finish selection"
-do
-    if [[ "$selected_driver" == "Finish selection" ]]; then
-        break
+# Step 3: Link all packages
+echo "Linking all packages..."
+for package in packages/*; do
+    if [ -d "$package" ]; then
+        echo "Linking $package..."
+        cd "$package"
+        yarn link
+        cd "$SCRIPT_DIR"
     fi
-    selected_drivers+=("$selected_driver")
-    echo "Selected drivers: ${selected_drivers[*]}"
 done
 
-# Step 6-7: Run yarn link and yarn install in packages/cubejs-<pkg>
-for driver in "${selected_drivers[@]}"
-do
-    echo "Linking and installing dependencies for $driver..."
-    cd "packages/cubejs-$driver"
-    yarn link
-    yarn install
-    cd ../..
+# Step 4: Ask for application name and database type
+read -p "Enter the application name: " APP_NAME
+
+# Get available database types from packages directory
+db_types=()
+for package in packages/cubejs-*-driver; do
+    if [ -d "$package" ]; then
+        db_name=$(basename "$package" | sed 's/cubejs-\(.*\)-driver/\1/')
+        db_types+=("$db_name")
+    fi
 done
 
-# Step 8: Run yarn link @cubejs-backend/<pkg> in packages/cubejs-server-core
-cd packages/cubejs-server-core
-for driver in "${selected_drivers[@]}"
+echo "Available database types:"
+PS3='Please select the database type: '
+select DB_TYPE in "${db_types[@]}"
 do
-    echo "Linking @cubejs-backend/$driver in packages/cubejs-server-core..."
```

Comment:
Are you sure linking drivers in server-core is no longer needed for proper working?

Reply:
I think so. This script is linking all the packages together.